docs/protocol/security.md

```diff
@@ -1,86 +1,235 @@
+### Passwords
+*
+ Be sure to make the password long enough, combining both registers letters, numbers and special characters. An 11-character password increases the bruteforce attack time to normally impossible limit but plese take a look on [passwords brutforce time illustrating table:](https://www.techrepublic.com/wp-content/uploads/2022/03/tr8423-Figure-A-How-an-8-character-password-could-be-cracked-in-just-a-few-minutes-770x404.jpg).
+*
+ Avoid using common or easily guessed combinations of words or letters, as well as parts of your online data or reusing any of your existing passwords.
 
 Weak password example:
 ```
-PickleRick!
+S0methin9C00l!
 ```
 Strong password example:
 ```
-hvDn@Hy#Gp5@jC*WH7
+^p$O_~a!4h{G'9C*
 ```
+*
+ Consider using password manager to securely store and generate complex passwords.
+There are plenty of password managers, make sure they store data in encrypted form, have strong password generators, good reviews, history of updates and work on their project. Use ratings, DYOR etc. At a quick glance (ignore the order) it's the likes of KeePass, Bitwarden, 1Password, LastPass, Dashlane, Keeper etc.
+*
+ Do not share your 12-word seed phrase or private keys
+*
+ A hardware wallet is by far the most secure way to store your private keys. Unlike online exchanges and wallets, hardware wallets store keys offline and protect from potential malware or hacking attempts.
+The most popular hardware wallets are [Ledger](https://www.ledger.com/) and [Tresor](https://trezor.io/).
+*
+ Backup your wallets: store the backups securely offline or in encrypted cloud storage. This ensures hat you can recover your funds in case of device loss, damage, or theft. There are also many encrypted cloud storage services available, like Tresorit, pCloud, Sync.com, SpiderOak, Mega (order does not matter). But DYOR.
+*
+ Be cautious of public Wi-Fi
+*
+ Keep software up to date
+*
+ If possible, enable 2FA (two-factor authentication) for all your cryptocurrency accounts and wallets. This provides an extra layer of security by requiring a second verification step, such as a code from a mobile app or hardware token, in addition to a password.
+*
+ Secure your devices! And here's a little basic guide for servers and home PCs in section #3.
+
+# 2. Community security
+*
+ Remember: Subspace team members will never send you direct messages first. If this happens to you, ignore the message, block the sender, and report the incident to our ["scam report" channel](https://discord.com/channels/864285291518361610/982023010405990400)
+*
+ Be ware of fishing attacks.
+Do not click on any links on our forum, Discord, or Telegram unless these links are posted by one of team members (ambassadors or staff). Scammers may create fake links looking like legit websites, so the easiest way to stay safe is to avoid such messages at all.
+
+# 3. Basic server securing
+
+*let's assume you just got a new server*
```

Comment:
I'd suggest to add a note about why it's needed, instead of this message and `etc ... check every SSD`